cmd/kubeadm/app/cmd/reset.go

```diff
@@ -97,7 +97,7 @@ func (r *Reset) Run(out io.Writer) error {
 
 	// Try to unmount mounted directories under /var/lib/kubelet in order to be able to remove the /var/lib/kubelet directory later
 	fmt.Printf("[reset] Unmounting mounted directories in %q\n", "/var/lib/kubelet")
-	umountDirsCmd := "cat /proc/mounts | awk '{print $2}' | grep '/var/lib/kubelet' | xargs -r umount"
+	umountDirsCmd := `awk '$2 ~ /\/var\/lib\/kubelet/ {print $2}' /proc/mounts | xargs -r umount`
```

Comment:
I don't know of any version issues with this particular usage, at least not within gawk. I would prefer the change as the back-ticking makes it hard to read and easy to get wrong. For instance, I first mistook the last forward-slash as part of the path and was going to call that out to be escaped. But I will also LGTM this if you feel strongly against it as it is still an improvement.